cli/cli.ts

```diff
@@ -3751,6 +3751,9 @@ function buildCoreAsync(buildOpts: BuildCoreOptions): Promise<pxtc.CompileResult
                 }
             }
 
+            if (pxt.appTarget.compile.switches.time)
+                console.log(compileResult.times)
```

Comment:
you can use "pxt.debug"

Reply:
pxt.debug get lost in all other noise, I guess pxt.log would be good, but need to stringify first

Reply:
Not worth it.